tests/integration/nodes/images/test_image_edit.py

```diff
@@ -0,0 +1,360 @@
+import io
+import uuid
+
+import pytest
+
+from dynamiq import Workflow, connections
+from dynamiq.callbacks import TracingCallbackHandler
+from dynamiq.callbacks.tracing import RunStatus, RunType
+from dynamiq.flows import Flow
+from dynamiq.nodes import NodeGroup
+from dynamiq.nodes.images import ImageEdit, ImageResponseFormat, ImageSize
+from dynamiq.runnables import RunnableConfig, RunnableStatus
+
+
+@pytest.fixture
+def mock_mask_file(mock_image_bytes):
+    """Create a test mask file."""
+    mask_file = io.BytesIO(mock_image_bytes)
+    mask_file.name = "test_mask.png"
+    mask_file.seek(0)
+    return mask_file
+
+
+@pytest.mark.parametrize(
+    ("response_format", "expected_content_type"),
+    [
+        (None, "url"),
+        (ImageResponseFormat.B64_JSON, "b64_json"),
+    ],
+)
+def test_image_edit_response_formats(
+    mock_image_file,
+    mock_image_edit_executor,
+    mock_httpx_get,
+    mock_image_url,
+    response_format,
+    expected_content_type,
+):
+    """Test image editing with different response formats."""
+    openai_connection = connections.OpenAI(id=str(uuid.uuid4()), api_key="test-api-key")
+
+    node = ImageEdit(
+        name="Image Editor",
+        model="gpt-image-1",
+        connection=openai_connection,
+        size=ImageSize.SIZE_1024x1024,
+        response_format=response_format,
+    )
+
+    wf = Workflow(
+        id=str(uuid.uuid4()),
+        flow=Flow(nodes=[node]),
+    )
+
+    input_data = {
+        "prompt": "Add a sunset background",
+        "image": mock_image_file,
+    }
+
+    response = wf.run(
+        input_data=input_data,
+        config=RunnableConfig(callbacks=[]),
+    )
+
+    assert response.status == RunnableStatus.SUCCESS
+    node_output = response.output[node.id]
+    assert "content" in node_output["output"]
+    assert "files" in node_output["output"]
+    assert len(node_output["output"]["files"]) == 1
+    assert isinstance(node_output["output"]["files"][0], io.BytesIO)
+    assert node_output["output"]["created"] == 1234567890
```

Comment:
What a magic number here? Please use a constant/mock, so can understand what is the mock here and where it defined